### ISSUE TITLE
"support show_change_link with related modal" feature is missing since version 0.20.0

### ISSUE THREAD

**Issue by bloodwithmilk25** [fabiocaccamo/django-admin-interface#323]
[This feature](https://github.com/fabiocaccamo/django-admin-interface/pull/120) was added in version 0.15.6 via [this commit](https://github.com/fabiocaccamo/django-admin-interface/commit/5ed9fd75cc7389b6b173ecd99aff5c5aca23e9da)
It's been missing since version 0.20.0, it looks like some kind of mistake

**Comment by fabiocaccamo** [comment]
@bloodwithmilk25 you're right, it was definitely removed by mistake during some refactoring, thank you for reporting this.

**Comment by FraCata00** [comment]
HI @fabiocaccamo , i just add a PR to resolve that

**Comment by fabiocaccamo** [comment]
@FraCata00 thank you!

**Comment by FraCata00** [comment]
> @FraCata00 thank you!
Thank you!! for merging 🔥
